packages/ripple-nuxt-tide/lib/core/html-utilities.js

```diff
@@ -5,7 +5,9 @@ export default {
       '&gt;': '>',
       '&lt;': '<',
       '&apos;': "'",
-      '&nbsp;': ' '
+      '&nbsp;': ' ',
+      '&quot;': '"',
+      '&#039;': "'"
```

Comment:
Hi @MdNadimHossain can you please update this test for `&#039;`? https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/test/unit/html-utilities.test.js

Reply:
Done, added test to check these three - &apos;&quot;&#039;